### DISCUSSION TITLE
How to specify target peak range using addReproduciblePeakSet() function ?

### DISCUSSION THREAD

**Discussion by heomanism** [GreenleafLab/ArchR#1972] [Questions / Documentation] [answered]
Hello, Thank you to the people made ArchR !

I have a question for you.

If I have specific target peak range (ex. chr1:1010294-1020294), can I create a peakSet file for using my peak range list and ArchR object that has fragment file ?

When I see the manual and option, I can't find anything related with my requestion.

**Comment by rcorces** [comment, marked as answer]
https://www.archrproject.com/reference/addPeakSet.html

**Comment by heomanism** [comment]
Hello, Thank you for fast reply. 

However, I can't get peakType annotation in resutls.

Is there any way to get the peakType in ArchR using my own peakSet ?

![image](https://github.com/GreenleafLab/ArchR/assets/87968423/0911585c-5892-41a9-a244-6c26d3487ebc)

**Reply by rcorces** [reply]
you would need to use the hidden `.fastAnnoPeaks()` function
https://github.com/GreenleafLab/ArchR/blob/c61b0645d1482f80dcc24e25fbd915128c1b2500/R/ReproduciblePeakSet.R#L590-L639
